pkg/nuctl/test/function_test.go

```diff
@@ -125,6 +126,43 @@ func (suite *functionDeployTestSuite) TestDeploy() {
 	suite.Require().Contains(suite.outputBuffer.String(), "+gnirts siht esrever-")
 }
 
+func (suite *functionDeployTestSuite) TestInvokeWithBodyFromStdin() {
+	uniqueSuffix := "-" + xid.New().String()
+	functionName := "reverser" + uniqueSuffix
+	imageName := "nuclio/deploy-test" + uniqueSuffix
+	namedArgs := map[string]string{
+		"path":    path.Join(suite.GetFunctionsDir(), "common", "reverser", "python"),
+		"image":   imageName,
```

Comment:
Like when we talked, adding image here seems weird and makes things not work as expected...
Maybe we should remove that from all the tests? wdyt?

Reply:
I would leave it + add a todo for where it stuck
reason: it works on most cases, as expected

Just leaving the note here for other readers:
`spec.build.image` simply tell what should be the container image name (meaning, when building a nuclio function, it will name the function container image with that name)

Reply:
That being said, I've decided to _listen_ to your suggestion .. and delete those unneeded "image" where it doesn't belong.

The reason is that, when runtime != shell and we deploy from local using nuctl,
specifying an "image" to run from + image to run with will lead to an error on second deploy because the local path we deployed with is no longer available to us and those, we *should* not rebuild (as specifying `spec.build.image` forces us to do).

This issue should be tackled on other PR (or @sahare92 will), I can't promise we will support that as it seems like it is something we should error on